setup.py

```diff
@@ -13,7 +13,7 @@
     long_description=open("README.rst").read(),
     url="https://github.com/nion-software/niondata",
     packages=["nion.data", "nion.data.test"],
-    install_requires=['scipy', 'numpy>=1.16', 'nionutils'],
+    install_requires=['scipy', 'numpy>=1.17', 'nionutils'],
```

Comment:
Also requires a change in `test-requirements.txt`. I just made a patch to require numpy>=1.16. Can you include the update to 1.17 in `test-requirements.txt` also in your PR?